js/noms/src/version.js

```diff
@@ -4,4 +4,24 @@
 
 // @flow
 
-export default '7';
+const stable = '7';
+const next = '8';
+let useNext = false;
+
+export default {
```

Comment:
This is an anti pattern. Export all the functions instead. You might still want to `export default '7';`